### DISCUSSION TITLE
TS80P disassembly

### DISCUSSION THREAD

**Discussion by krikun98** [Ralim/IronOS#1237] [Q&A]
Hi. My TS80P is sometimes not recognizing the tips, and I'm mostly able to remedy the issue by poking inside the 3.5 mm. jack with some tweezers trying to bend back the contacts, but it's annoying and hardly reliable. I saw someone here disassembled theirs, is there a safe/non-destructive way to do that? I removed the collar and the (I assume) grounding screw, but didn't really figure out where to go from there. And is there a part number for the 3.5 mm. jack? I'd just like to swap it out.

**Comment by discip** [comment]
@krikun98
Here you go:
**https://github.com/Ralim/IronOS/issues/630#issuecomment-809748658**
